streams/dshr_strdata_mod.F90

```diff
@@ -778,7 +778,7 @@ subroutine shr_strdata_advance(sdat, ymd, tod, logunit, istr, timers, rc)
           ! fldbun_stream_ub to fldbun_stream_lb and read in new fldbun_stream_ub data
           ! ---------------------------------------------------------
 
-!          call t_barrierf(trim(lstr)//trim(timname)//'_readLBUB_BARRIER',mpicom)
+          ! call t_barrierf(trim(lstr)//trim(timname)//'_readLBUB_BARRIER',mpicom)
```

Comment:
remove?

Reply:
yes - if its just going to be commented out - lets remove it. done.